docs/application/dotnet/guides/nui/window.md

```diff
@@ -0,0 +1,335 @@
+
+[Additional windows](#additionalwindow) can be created (as needed) to display additional content.
+
+The window delivers [various events](#eventhandling) (e.g. key event and touch event) to the application.
+
+<a name="defaultwindow"></a>
+## Creating window
+
+When an application is created, the default window will be automatically created:
+ ```csharp
+ MyApplication myApp = new MyApplication();
+ ```
+
+By default, the default window is created in full screen size.
+
+It also allows to specify the initial size and position of the default window by passing them to the constructor of the application:
+
+```csharp
+public MyApplication(Size2D windowSize, Position2D windowPosition) : base(windowSize, windowPosition)
+{
+}
+
+MyApplication myApp = new MyApplication(new Size2D(1920, 1080), new Position2D(0, 0));
+```
+
+The instance of the default window can be retrieved:
+
+```csharp
+Window window = Window.Instance;
+```
+
+The window itself is just a blank container, and the views can be added to or removed from the window.
+
+```csharp
+// Create a new view
+View view = new View();
+
+// Add the view to the window
+window.Add(view);
+
+// Remove the view from the window
+window.Remove(view);
+```
+
+Below is a simple example of how to add content to the window:
+
+```csharp
+// Get the window instance
+Window window = Window.Instance;
+window.BackgroundColor = Color.White;
+
+// Create a TextLabel
+TextLabel title = new TextLabel("Hello World");
+title.Size2D = new Size2D(window.Size.Width, window.Size.Height);
+title.HorizontalAlignment = HorizontalAlignment.Center;
+title.VerticalAlignment = VerticalAlignment.Center;
+
+// Add the text to the window
+window.Add(title);
+```
+
+<img src="./media/new_window3.png">
+
+<a name="deleting"></a>
+## Deleting window
+
+A window can be deleted, which will also delete all the views inside the window. However, the window cannot be deleted.
+
+```csharp
+newWindow.Destroy();
+```
+
+
+
+<a name="resizing"></a>
+## Resizing window
+
+The size of a window can be retrieved, and the window can be resized.
+
+```csharp
+// Retrieve the current size of the window
+Size2D windowSize = window.WindowSize;
+
+// Resize the window by increasing its width
+windowSize.Width += 100;
+window.WindowSize = windowSize;
+```
+
+When the window size is changed, a [resize event](#resizing) will be emitted.
+
+<a name="moving"></a>
+## Moving window
+
+It is easy to retrieve the current position of a window and move the window to a new position.
+
+```csharp
+// Retrieve the current position of the window
+Position2D windowPosition = window.WindowPosition;
+
+// Moving the window to the right
+windowPosition.X += 100;
+window.WindowPosition = windowPosition;
+```
+
+<a name="showinghiding"></a>
+## Showing and hiding window
+
+Every window is shown after being created. It is possible to hide a window to make it invisible.
```

Comment:
When a window is created, by default it is in visible state. It is also possible to hide a window.

Please verify

Reply:
@mijin85cho,  thank you. I agree with your suggestion.